### ISSUE TITLE
setup as package

### ISSUE THREAD

**Issue by tacaswell** [MJCruickshank/SARfish#5]
Would a PR adding a project.toml (making this a pip-installable package) + entry points + docs + tests be welcome?

This is also the first step in getting this packagable for conda-forge.

**Comment by meghankane** [comment]
Yes, that would be really cool and appreciated if you have time to do that! 😄

**Comment by MJCruickshank** [comment]
Seconded - that would be amazing if you aren't too busy!

**Comment by meghankane** [comment]
and we'll make sure to review it faster next time

**Comment by giswqs** [comment]
I have a Python Cookiecutter package template that might be useful for this. It has mkdocs-material built in. 
https://github.com/giswqs/pypackage

Check out my Python packages built using the template:

- https://geemap.org/
- https://leafmap.org/

**Comment by tacaswell** [comment]
Sorry, I got way busier than I expected.  I'll still try to get to this when I have time maybe later this week or next weekend, but will not be upset in the slightest if someone else did it ;)

I was going to start with https://nsls-ii.github.io/scientific-python-cookiecutter/ + some modernizations (gha instead of travis, pyproject.toml instead of setup.py etc)
